src/cli/cli.ts

```diff
@@ -660,6 +660,10 @@ function printHelp(helpMode: string, exitCode: number = 0): void {
     printCommandHelp(helpMode);
   }
   else {
+    if (cli.currentCommandName) {
+      cli.error(`Command '${cli.currentCommandName}' not found. Please find a list of available commands below.`);
```

Comment:
The message we're throwing doesn't seem very clear to me. We do list commands but also command groups. So a message like this would sound more clear `Command '${cli.currentCommandName}' was not found. Below you can find the commands and command groups you can use. For detailed information on a command group, use 'm365 [command group] --help'.`